### PR TITLE
feat(opentrons-ai-client): add prompt button

### PR DESCRIPTION
**Pull request by koji**
# Overview
add prompt button component and use it in InputPrompt component.

Textarea's CSS will be updated in a following PR

https://github.com/Opentrons/opentrons/assets/474225/fbcdd0a8-3fe3-4bc9-b058-0d65b106e6f5

close AUTH-334

# Test Plan
- make -C opentrons-ai-client dev and localhost:5173
- make -C components dev and PromptButton

# Changelog
- add PromptButton component, test, its storybook(not working now)

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] add tests
- [x] add stories

**Comment by ncdiehl11** [comment]
Looks good, just a note that when 100% zoomed on Chrome, the text area appears to be cut off a bit. Also just confirming that the intended behavior is for the clicked prompt button is to remain darker after click
Screenshot 2024-04-23 at 9 43 57 AM

**Comment by koji** [comment]
> Looks good, just a note that when 100% zoomed on Chrome, the text area appears to be cut off a bit. Also just confirming that the intended behavior is for the clicked prompt button is to remain darker after click Screenshot 2024-04-23 at 9 43 57 AM

yeah, that happens because the height of SidePanel and this is a known issue.
Sue will update the entire application design after the external beta.

Oh thanks, the button's behavior you mentioned isn't expected. I will fix it.

File: opentrons-ai-client/src/organisms/PromptButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const usePromptSetValue = (): React.Dispatch<React.SetStateAction<string>> =>
+    React.useContext(setPromptContext)
+  const setPrompt = usePromptSetValue()
+
+  const handleClick = useCallback(() => {
+    const { prompt } = PROMPT_BY_NAME[buttonText]
+    setPrompt(prompt)
+  }, [setPrompt, buttonText])
+
+  return <PromptBtn onClick={handleClick}>{buttonText}</PromptBtn>
```
Comment (ncdiehl11) [inline opentrons-ai-client/src/organisms/PromptButton/index.tsx:46]: LGTM once the clicked style of PromptBtn is fixed! 👍

**Review by ncdiehl11 — APPROVED** [review]

**Pull request merged**